driver/bpf/fillers.h

```diff
@@ -4253,6 +4253,143 @@ FILLER(sys_recvmsg_x_2, true) {
 	return res;
 }
 
+FILLER(sys_recvmmsg_x, true) {
+	const struct iovec *iov;
+	struct mmsghdr mmh;
+	unsigned long iovcnt;
+	unsigned long val;
+	int res;
+	int fd;
+
+	/* Parameter 1: res (type: PT_ERRNO) */
+	long retval = bpf_syscall_get_retval(data->ctx);
+
+	/* If the syscall fails we are not able to collect reliable params
+	 * so we return empty ones.
+	 */
+	if(retval < 0) {
+		res = bpf_push_s64_to_ring(data, retval);
+		CHECK_RES(res);
+
+		/* Parameter 2: fd (type: PT_FD) */
+		res = bpf_push_empty_param(data);
+		CHECK_RES(res);
+
+		/* Parameter 3: size (type: PT_UINT32) */
+		res = bpf_push_u32_to_ring(data, 0);
+		CHECK_RES(res);
+
+		/* Parameter 4: data (type: PT_BYTEBUF) */
+		res = bpf_push_empty_param(data);
+		CHECK_RES(res);
+
+		/* Parameter 5: tuple (type: PT_SOCKTUPLE) */
+		res = bpf_push_empty_param(data);
+		CHECK_RES(res);
+
+		/* Parameter 6: msg_control (type: PT_BYTEBUF) */
+		return bpf_push_empty_param(data);
+	}
+
+	fd = bpf_syscall_get_argument(data, 0);
+
+	/*
+	 * Retrieve the message header
+	 */
+	val = bpf_syscall_get_argument(data, 1);
+	if(bpf_probe_read_user(&mmh, sizeof(mmh), (void *)val))
+		return PPM_FAILURE_INVALID_USER_MEMORY;
+
```

Comment:
can we use the usual comments like `/* Parameter 1: res (type: PT_ERRNO) */` just to understand what we are pushing